assets/src/edit-story/components/panels/karma/sizePosition.karma.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+import { useStory } from '../../../app/story';
+
+describe('Size & Position Panel', () => {
+  let fixture;
+  let panel;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    await fixture.render();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  const selectTarget = async (target) => {
+    const { x, y, width, height } = target.getBoundingClientRect();
+    await fixture.events.keyboard.down('Shift');
+    await fixture.events.mouse.click(x + width / 2, y + height / 2);
+    await fixture.events.keyboard.up('Shift');
+  };
+
+  const getSelection = async (first = true) => {
+    const storyContext = await fixture.renderHook(() => useStory());
+    return first
+      ? storyContext.state.selectedElements[0]
+      : storyContext.state.selectedElements;
+  };
```

Comment:
You could skip the `first` argument and just _destruct-or-not_ when using the return:

```js
const updatedElements = await getSelection();
const [updatedElement] = await getSelection();
```

I think that's a bit clearer – and less confusing when you come back to it in 6 months.